src/shrpx_config_test.cc

```diff
@@ -33,31 +33,32 @@
 #include <CUnit/CUnit.h>
 
 #include "shrpx_config.h"
+#include "util.h"
 
 namespace shrpx {
 
 void test_shrpx_config_parse_config_str_list(void) {
```

Comment:
Since parse_config_str_list is now defined in util.cc, this test code should be in util_test.cc, renaming as `test_util_parse_config_str_list`